src/wp-admin/security.php

```diff
@@ -22,7 +22,7 @@
     <h2><?php _e( '"Security First"' ); ?></h2>
     <p><?php _e( "Security is important to business and it’s important to us here at ClassicPress. By bringing security forward to a place of greater prominence within the admin interface, we create a more streamlined experience for both users and developers." ); ?></p>
     <p><?php _e( "As ClassicPress continues to evolve, the Security page will become the hub for all security features for ClassicPress core and 3<sup>rd</sup> party plugins that choose to support it." ); ?></p>
-    <p><?php _e( "Watch this page and the ClassicPress Security forum for more news as development continues. If you’d like to contribute in this area, please connect with us <a href=\"#\">here</a>." ); ?></p>
+    <p><?php _e( "Watch this page and the ClassicPress Security forum for more news as development continues. If you’d like to contribute in this area, please connect with us <a href=\"https://forums.classicpress.net/security\" target=\"_blank\">here</a>." ); ?></p>
```

Comment:
It looks like you can use https://forums.classicpress.net/c/team-discussions/classicpress-security to avoid an extra redirect.

Also the usual way to translate strings like this is as follows:

```
printf(
    /* translators: link to security subforum */
    __( 'Watch this page and etc more info <a href="%s">on our security subforum</a>.' ),
    'https://forums.classicpress.net/c/team-discussions/classicpress-security'
);
```

Reply:
That's the correct canonical URL.
As for `printf` - OK, but does glotpress check the conversion specifiers match?

Reply:
I'm not sure.  Maybe @Mte90 knows.

Reply:
For glotpress there will be no replace for string, it will use the content inside `__()` and extract the comment inside printf as description for the localizers